lib/biokbase/catalog/controller.py

```diff
@@ -1166,7 +1166,7 @@ def has_permission(self, username, token, owners):
         return False
 
     def is_admin(self, username, token):
-        r = requests.get(self.auth_url + '/api/V2/me', headers={'Authorization': token})
+        r = requests.get(self.auth_api + '/api/V2/me', headers={'Authorization': token})
```

Comment:
It's currently set to 'auth_service_url=https://ci.kbase.us/services/auth/api/legacy/KBase/Sessions/Login'

Reply:
So this won't work.